docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,590 @@
+
+### Configuring your Xcode project
+
+<Step label="1">
+  Open your project on Xcode and from the project navigator, select your project and then select
+  your app target under `TARGETS`. In `Build Settings`, using the search bar, search for
+  `ENABLE_USER_SCRIPT_SANDBOXING`. If it is not already, set its value to `No`. This is needed to
+  properly switch between the Debug and Release version of the [Hermes
+  engine](https://github.com/facebook/hermes/blob/main/README.md) that is shipped with React Native.
+</Step>
+
+<Step label="2">
+
+Now switch to the `Build Phases` tab and add a new `Run Script Phase` before the `[CP] Embed Pods Frameworks` phase. This script will bundle your JavaScript code and assets into the iOS application.
+
+```sh Build React Native code and image
+if [[ -f "$PODS_ROOT/../.xcode.env" ]]; then
+  source "$PODS_ROOT/../.xcode.env"
+fi
+if [[ -f "$PODS_ROOT/../.xcode.env.local" ]]; then
+  source "$PODS_ROOT/../.xcode.env.local"
+fi
+
+# The project root by default is one level up from the ios directory
+export PROJECT_ROOT="$PROJECT_DIR"/..
+
+if [[ "$CONFIGURATION" = *Debug* ]]; then
+  export SKIP_BUNDLING=1
+fi
+if [[ -z "$ENTRY_FILE" ]]; then
+  # Set the entry JS file using the bundler's entry resolution.
+  export ENTRY_FILE="$("$NODE_BINARY" -e "require('expo/scripts/resolveAppEntry')" "$PROJECT_ROOT" ios absolute | tail -n 1)"
+fi
+
+if [[ -z "$CLI_PATH" ]]; then
+  # Use Expo CLI
+  export CLI_PATH="$("$NODE_BINARY" --print "require.resolve('@expo/cli', { paths: [require.resolve('expo/package.json')] })")"
+fi
+if [[ -z "$BUNDLE_COMMAND" ]]; then
+  # Default Expo CLI command for bundling
+  export BUNDLE_COMMAND="export:embed"
+fi
+
+# Source .xcode.env.updates if it exists to allow
+# SKIP_BUNDLING to be unset if needed
+if [[ -f "$PODS_ROOT/../.xcode.env.updates" ]]; then
+  source "$PODS_ROOT/../.xcode.env.updates"
+fi
+# Source local changes to allow overrides
+# if needed
+if [[ -f "$PODS_ROOT/../.xcode.env.local" ]]; then
+  source "$PODS_ROOT/../.xcode.env.local"
+fi
+
+`"$NODE_BINARY" --print "require('path').dirname(require.resolve('react-native/package.json')) + '/scripts/react-native-xcode.sh'"`
+```
+
+Next time when you build your app for Release, the React Native code will be bundled using Expo CLI and embedded into the app.
+
+</Step>
+
+<Step label="3">
+Edit your `Info.plist` file and make sure to add the `UIViewControllerBasedStatusBarAppearance` key with value of `NO`, this is needed to ensure that the status bar is properly managed by React Native.
+
+```diff
++<key>UIViewControllerBasedStatusBarAppearance</key>
++<string>NO</string>
+```
+
+</Step>
+
+### Integrating with your code
+
+We now need to add some native code in order to start the React Native runtime and tell it to render our React components.
+
+#### Create the ReactViewController
+
+Create a new file called `ReactViewController.swift`, this will be the ViewController that loads a React Native view as it's `view`.
```

Comment:
```suggestion
Create a new file called **ReactViewController.swift**, this will be the `ViewController` that loads a React Native view as its `view`.
```